queries/generators/netherlands.rq

```diff
@@ -24,7 +24,12 @@ WHERE {
 
   MINUS { ?org wdt:P576 [] }
   MINUS { ?org wdt:P1366 [] }
-  MINUS { ?org wdt:P582 [] }
+  MINUS { ?org wdt:P582 [] .
```

Comment:
This appears to be incompatible with the MINUS-sections for the municipalities.

Reply:
There ends up still being 11 ministries if one does not remove the two MINUS-sections below this one.

Reply:
Strange, this query gets 12: https://w.wiki/4xxj

Reply:
Which 11 do you see? When I run this query in WDQS I get 12.

Reply:
(Perhaps you are confused by the 410 instead of 411, but that is an unrelated error because we get 344 municipalities right now.)